packages/@ama-sdk/schematics/package.json

```
@@ -113,7 +113,7 @@
     "type-fest": "^4.10.2",
     "typescript": "~5.4.2"
   },
-  "openApiSupportedVersion": "~6.6.0",
+  "openApiSupportedVersion": "~7.4.0",
```

```suggestion
  "openApiSupportedVersion": ">=6.0.0 <7.5.0",
```
because we still support 6.6.0 no ?